Resources/public/js/views/ez-userprofileview.js

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-userprofileview', function (Y) {
+    'use strict';
+    /**
+     * Provides the User Profile View class
+     *
+     * @module ez-userprofileview
+     */
+    Y.namespace('eZ');
+
+    var CLASS_USER_PROFILE_HIDDEN = 'is-user-profile-hidden',
+        SELECTOR_USER_NAME = '.ez-user-profile-name',
+        SELECTOR_USER_ROLE = '.ez-user-profile-role',
+        SELECTOR_USER_IMAGE = '.ez-user-profile-image';
+
+    /**
+     * The User Profile View
+     *
+     * @namespace eZ
+     * @class UserProfileView
+     * @constructor
+     * @extends eZ.TemplateBasedView
+     */
+    Y.eZ.UserProfileView = Y.Base.create('userProfileView', Y.eZ.TemplateBasedView, [], {
+        events: {
+            '.ez-user-profile': {
+                'tap': '_toggleUserMenu'
+            }
+        },
+
+        initializer: function () {
+           this.after('*:userInfoChange', this._setUserInfo, this);
+        },
+
+        /**
+        * Renders the user profile view
+        *
+        * @method render
+        * @return {eZ.UserProfileView} the view itself
+        */
+        render: function () {
+            this.get('container')
+                .setHTML(this.template())
+                .addClass(CLASS_USER_PROFILE_HIDDEN);
+
+            this.fire('getUserInfo');
```

Comment:
it's weird to fire an event like that and in `render`. Normally the data you need to render the view should be provided by default without the need to ask anything. So basically, the navigation hub view service load method should more or less do what is currently done when handling this event.